Client/Frontend/Login Management/LoginDetailTableViewCell.swift

```diff
@@ -59,22 +59,22 @@ class LoginDetailTableViewCell: ThemedTableViewCell {
     // Exposing this label as internal/public causes the Xcode 7.2.1 compiler optimizer to
     // produce a EX_BAD_ACCESS error when dequeuing the cell. For now, this label is made private
     // and the text property is exposed using a get/set property below.
-    fileprivate lazy var highlightedLabel: UILabel = {
```

Comment:
I'd suggest to use to multiline comments here to minimise the changeset a bit, but no blocker